.eslintrc

```diff
@@ -223,7 +223,6 @@
     //
     // React specific linting rules for ESLint
     //
-    "react/display-name": 1, // Prevent missing displayName in a React component definition
```

Comment:
It relies on the name of the variable assigned when instantiating the component. As long as we have good conventions of using good names, then yea I think you're right, maybe we could do without this. Ok cool! :+1: